include/mim_solvers/csqp.hpp

```diff
@@ -190,6 +191,8 @@ class SolverCSQP : public SolverDDP {
   double get_reset_rho() const { return reset_rho_; };
   double get_rho_min() const { return rho_min_; };
   double get_rho_max() const { return rho_max_; };
+  double get_max_solve_time() const { return max_solve_time_; };
```

Comment:
```suggestion
  /// The maximum resolution time in seconds.
  double get_max_solve_time() const { return max_solve_time_; };
```

Mind the triple `/` so that doxygen sees it.

Reply:
The problem is that none of the functions are documented. This is why I didn't add a comment